frame/vesting/src/lib.rs

```diff
@@ -473,12 +462,14 @@ mod tests {
 	}
 	parameter_types! {
 		pub const MinVestedTransfer: u64 = 256 * 2;
+		pub const MaxLocks: u32 = 10;
 	}
 	impl Trait for Test {
 		type Event = ();
 		type Currency = Balances;
 		type BlockNumberToBalance = Identity;
 		type MinVestedTransfer = MinVestedTransfer;
+		type MaxLocks = MaxLocks;
```

Comment:
```suggestion
		type MaxLocks = ();
```

I think implementation of `Get` for `()` is underrated in our repo :D.